docs/docs/anatomy.md

```diff
@@ -0,0 +1,254 @@
+---
+sidebar_position: 3
+---
+
+# Anatomy of an Equinox Service
+
+An Equinox service is composed of a few modules (namespaces).
+
+- **Stream**: tells us which category the service writes to and how the identity
+  of the stream is composed (and the reverse operations for when running Reaction logic)
+- **Events**: tells us which events the service writes
```

Comment:
and how they are to be mapped to/from the stored form (e.g. applying wrapper types over primitives)

but only if you an convey it more tersely
the aim being to sell why they are needed and not overkill in a one liner